schemas/common_definitions.json

```diff
@@ -42,12 +43,25 @@
         "type": "string",
         "description": "The text that is used for the 'Hide guidance' link."
       },
-      "content": {
-        "$ref": "common_definitions.json#/content"
+      "contents": {
+        "$ref": "common_definitions.json#/contents"
       }
     }
   },
   "content": {
+    "description": "Allows customisation of guidance text.",
+    "type": "object",
+    "properties": {
+      "title": {
+        "type": "string",
+        "description": "The H1 title on the page."
```

Comment:
The H1 is implementation specific, it would be better to say something like "main" or "top-level" heading on the page.